contracts/contracts/interfaces/IFraxETHMinter.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IFraxETHMinter {
+    function submitAndDeposit(address recipient)
+        external
+        payable
+        returns (uint256 shares);
+
+    function submit() external payable;
```

Comment:
I don't think this `submit` is being used, and can be removed?

Reply:
That's true, will remove it